client/dashboard/sites/site/environment-switcher.tsx

```diff
@@ -231,6 +233,10 @@ const EnvironmentSwitcher = ( { site }: { site: Site } ) => {
 			productionSite &&
 			stagingSiteId
 		) {
+			createSuccessNotice( __( 'Staging site deleted.' ), {
+				type: 'snackbar',
+				explicitDismiss: true,
+			} );
```

Comment:
I think that since the user probably won't care much about their deleted staging site anymore and the process is quite quick, we could leave the notice to disappear on its own.

```suggestion
			createSuccessNotice( __( 'Staging site deleted.' ), {
				type: 'snackbar',
			} );
```

Reply:
Adjusted in https://github.com/Automattic/wp-calypso/pull/105671/commits/c0185ad24b0d8da07c7eab684c46d49a0f1b1144